assessment-api/open-api-specification.yml

```diff
@@ -3,12 +3,29 @@ info:
   title: 'laa-ccms-assessment-api'
   version: '1.0.0'
 paths:
-  /assessment:
+  /assessments:
     get:
       tags:
         - 'assessment'
-      summary: 'Assessment'
-      operationId: 'assessment'
+      summary: 'get assessments'
+      operationId: 'getAssessments'
+      parameters:
+        - $ref: '#/components/parameters/assessmentParameter'
```

Comment:
Should this not be assessmentTypeParameter?

Reply:
will amend